FirebaseRTC/public/patWaitingRoom.html

```diff
@@ -23,10 +23,10 @@
 
     <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
   <link rel="stylesheet" type="text/css" href="patWaitingRoom.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/font-awesome.css">
+    <link rel="stylesheet" type="text/css" href="/assets/csslatemo-art-factory.css">
```

Comment:
"/temp" was lost from "css/templatemo-art-factory.css" Be careful with find and replace function.

Reply:
The issue is fixed in the new commit.

Reply:
Line 28 is still incorrect. 
Should be:
<link rel="stylesheet" type="text/css" href="/assets/css/templatemo-art-factory.css">

Reply:
<link rel="stylesheet" type="text/css" href="/assets/css/templatemo-art-factory.css">